### PR TITLE
[doc] export OPENMS_DATA_PATH for .deb package

### PR DESCRIPTION
**Pull request by oliveralka**
Documentation: export OPENMS_DATA_PATH for .deb package

### CODE REVIEW

File: doc/doxygen/install/install-linux-bin.doxygen [hunk 1]
```diff
@@ -7,10 +7,15 @@
   @section package_managers_linux Install via package managers
   Currently there are packaged versions of %OpenMS provided for Fedora, OpenSUSE, Debian, and Ubuntu <a href="https://pkgs.org/search/openms">available in the corresponding package managers</a> of these distributions (they might require a lookup in thirdparty "science" repositories).
   For other GNU/Linux distributions or to obtain the most recent version of the library you need to build your own %OpenMS.
+  
+  To ensure the tool functionality please add the OPENMS_DATA_PATH variable. 
+  \code
+  export OPENMS_DATA_PATH=/usr/shared/OpenMS
```
Comment (jpfeuffer) [inline doc/doxygen/install/install-linux-bin.doxygen]: Is it share or shared?

Comment (oliveralka) [reply doc/doxygen/install/install-linux-bin.doxygen]: share is right

**Comment by timosachsenberg** [comment]
develop or release branch?

**Comment by jpfeuffer** [comment]
Release

**Comment by timosachsenberg** [comment]
will put on release branch

**Comment by timosachsenberg** [comment]
is /usr/bin/SEARCHENGINES kind of a misnomer - we now not only ship search engines?
Should we update the docu that the user needs to set the path to the individual subfolders?

**Comment by jpfeuffer** [comment]
Yes I agree with both. Is a relic from old times. Should be renamed and documented.

**Comment by timosachsenberg** [comment]
k should this be put into this release?

**Comment by oliveralka** [comment]
release +1

**Comment by jpfeuffer** [comment]
How about adding the issue about Searchengines here?

**Comment by jpfeuffer** [comment]
How about adding the issue about Searchengines here?

**Comment by jpfeuffer** [comment]
Naming issue would be here:
https://github.com/OpenMS/OpenMS/blob/develop/cmake/install_macros.cmake#L143
Maybe we should change it to share/OpenMS/THIRDPARTY like on Windows.

**Comment by timosachsenberg** [comment]
will this make debian guys angry? putting binaries there?

**Comment by jpfeuffer** [comment]
Maybe. Do they even ship the thirdparties?

**Comment by jpfeuffer** [comment]
Also affects macOS. Uses same mechanism.

**Comment by hroest** [comment]
my preference would be to ship OpenMS "batteries included" but from a debian perspective that does not make sense. I am pretty sure they would like to break it all down into individual packages and they already do have some in debian:

- https://packages.debian.org/sid/comet-ms
- https://packages.debian.org/sid/tandem-mass

so if we go through them, we would have to package all the third party tools / get somebody to package it. We then have the issue of (i) slowing it all down and (ii) not having control over the versions but of course that would be "right way to do it" 

> will this make debian guys angry? putting binaries there?

we will just need to tell them and I assume they will delete them. There is anyways a policy in debian that you cannot ship binaries, all binaries have to be compiled from source and have to be under an open source license.

**Comment by jpfeuffer** [comment]
I agree. They will change it if it does not agree with their policies. Completely modularizing indeed will take some time. Anyway, this pr is mainly about the naming. Thirdparties are shipped in any case.

**Comment by oliveralka** [comment]
@jpfeuffer - I'm not entirely sure of the consensus related to this pull request. Any further changes necessary?

**Comment by jpfeuffer** [comment]
This could be changed.

_Sent from my Xperia Z5 Compact using [FastHub-Libre](https://github.com/thermatk/FastHub-Libre/)_

**Comment by oliveralka** [comment]
Closed merge #3122

**Pull request closed without merging**